components/ProgressBar.js

```diff
@@ -16,27 +17,42 @@ export default function Progress({
     setReset(0);
     onReset(reset);
   }
-  function saveClick() {}
+  function handleSave(e) {
+    e.preventDefault();
+    localStorage.setItem("name", JSON.stringify(savedpercent));
+    localStorage.setItem("budget", JSON.stringify(savedbudget));
+  }
+
   const addFive = budget + addvalue + minusvalue + "€";
```

Comment:
You shouldn't turn your number into a string when you want to calculate with it...

Reply:
I don't want to calculate it, but simply store it in localstorage.